### PR TITLE
CONN-1278: Resolved rebase conflict, completed X12 Adapter code

### PR DESCRIPTION
**Pull request by sailajaa**

### CODE REVIEW

File: Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeImpl.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import gov.hhs.fha.nhinc.messaging.server.BaseService;
+import javax.xml.ws.WebServiceContext;
+
+/**
+ *
+ * @author sadusumilli
+ */
+public class AdapterCORE_X12DSRealTimeImpl extends BaseService {
+
+    private OutboundCORE_X12DSRealTime outboundCOREX12DSRealTime;
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeImpl.java]: Remove this variable and the constructor.  Not needed here.

File: Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeSecured.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+/**
+ *
+ * @author sadusumilli
+ */
+public class AdapterCORE_X12DSRealTimeSecured implements gov.hhs.fha.nhinc.adaptercoresecured.AdapterCORETransactionSecuredPortType {
+
+    @Resource
+    private WebServiceContext context;
+    private OutboundCORE_X12DSRealTime outboundCORE_X12DSRealTime;
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeSecured.java]: Outbound isn't used here either.  Remove the setter as well.

File: Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeUnsecured.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+/**
+ *
+ * @author sadusumilli
+ */
+public class AdapterCORE_X12DSRealTimeUnsecured implements gov.hhs.fha.nhinc.adaptercore.AdapterCORETransactionPortType {
+
+    @Resource
+    private WebServiceContext context;
+    private OutboundCORE_X12DSRealTime outboundCORE_X12DSRealTime;
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeUnsecured.java]: remove outbound.

File: Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/webapp/WEB-INF/cxf-servlet.xml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    <!-- Adapter X12 Document Submission -->
+    <cxf:bus>
+        <cxf:features>
+            <p:policies />
+        </cxf:features>
+    </cxf:bus>
+
+    <jaxws:endpoint xmlns:tns="urn:gov:hhs:fha:nhinc:adaptercoresecured" id="AdapterDocSubmissionCoreSecured"
+        address="/CORETransaction/AdapterCORETransactionSecured" serviceName="tns:AdapterCORETransactionSecured" endpointName="tns:AdapterCORETransactionSecured_Port"
+        implementor="#AdapterCORESecured" wsdlLocation="classpath:wsdl/AdapterCORERule2.2.0Secured.wsdl">
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/webapp/WEB-INF/cxf-servlet.xml]: The wsdls were updated and these will need to be updated (both secure and unsecure.

File: Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeImpl.java [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+    /**
+     *
+     * @param adapterCOREEnvelopeRealTimeRequestType
+     * @param context
+     * @return
+     */
+    public AdapterCOREEnvelopeRealTimeResponseType realTimeTransaction(AdapterCOREEnvelopeRealTimeRequestType adapterCOREEnvelopeRealTimeRequestType, WebServiceContext context) {
+        AdapterCOREEnvelopeRealTimeResponseType oResponse = new AdapterCOREEnvelopeRealTimeResponseType();
+        oResponse.setCOREEnvelopeRealTimeResponse(new AdapterCORE_X12DSRealTimeOrchImpl().realTimeRequest(adapterCOREEnvelopeRealTimeRequestType.getCOREEnvelopeRealTimeRequest(), adapterCOREEnvelopeRealTimeRequestType.getAssertion()));
```
Comment (jsmith2) [inline Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/adapter/AdapterCORE_X12DSRealTimeImpl.java]: still need to build the assertion here, need to add the line: AssertionType oAssertion = getAssertion(context, adapterCOREEnvelopeRealTimeRequestType.getAssertions())

(you could also shorten that variable name to just adapterRequest, it's a bit long)

**Comment by christophermay07** [comment]
retest this please

**Pull request merged**